iothub/service/src/ServiceClient.cs

```diff
@@ -52,6 +59,82 @@ public static ServiceClient CreateFromConnectionString(string connectionString,
             return CreateFromConnectionString(connectionString, TransportType.Amqp, options);
         }
 
+#if !NET451
+
+        /// <summary>
+        /// Creates a <see cref="ServiceClient"/> using AAD token credentials and the specified transport type.
```

Comment:
All caps for Acronyms like AAD and SAS. Here, I'd l actually like AAD spelled out to "Azure Active Directory" rather than assuming customers know what AAD is.

Reply:
AAD is an acronym, so all caps. SAS is also an acronym, so all caps.